res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -14,6 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+:root {
+    --RoomHeader-indicator-dot-size: 8px;
+    --RoomHeader-indicator-dot-offset: -3px;
+    --RoomHeader-indicator-pulseColor: $alert;
+}
```

Comment:
I am not really satisfied with this type of "solution". Instead there should be one common `Indicator` component, which could be customized with custom properties for each case. I guess this should be a better way of using the custom properties, rather than adding ones for each case, so that cascading works effectively, keeping the number of properties as low as possible.